mne/decoding/receptive_field.py

```diff
@@ -294,7 +324,33 @@ def _check_dimensions(self, X, y, predict=False):
         return X, y
 
 
-def _delay_time_series(X, tmin, tmax, sfreq, newaxis=0, axis=0):
+def _pad_time_series(X, n_delays, fill_mean=True):
```

Comment:
This one is a bit different because it isn't trying to get to a next fast FFT length, just trying to prevent overlapping during convolution. So I don't think so.